shared/tsconfig.base.json

```diff
@@ -9,9 +9,15 @@
     "jsx": "preserve",
     "module": "commonjs",
     "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```

Comment:
Does this override `strict`?

Reply:
strict is a bag of flags. we aren't anywhere close to supporting any
```Enable all strict type checking options. 
Enabling --strict enables --noImplicitAny, --noImplicitThis, --alwaysStrict, --strictBindCallApply, --strictNullChecks, --strictFunctionTypes and --strictPropertyInitialization.```

Reply:
the goal is to turn them all on eventually but we can't do them all at the same time and get it working across all files at the same time